3.7-rc/alpine3.7/Dockerfile

```diff
@@ -47,12 +47,12 @@ RUN set -ex \
 		libc-dev \
 		libffi-dev \
 		libnsl-dev \
+		openssl \
+		openssl-dev \
```

Comment:
I realize this now makes the openssl libs "unsorted", but the openssl builds should go away soon. Alpine 3.4 is being deprecated and Alpine 3.8's libressl should be supported by Python 3.7.